libs/redisconsumer/consumer_test.go

```diff
@@ -0,0 +1,89 @@
+package redisconsumer
+
+import (
+	"testing"
+
+	must "github.com/stretchr/testify/require"
+)
+
+func TestChunkMessagesEmpty(t *testing.T) {
+	var testMessages []interface{}
+
+	testChunks := chunkMessages(500, testMessages)
+	must.Equal(t, 0, len(testChunks))
+}
+
+func TestChunkMessagesSingleChunk(t *testing.T) {
+	var testMessages []interface{}
+
+	for i := 0; i < 500; i++ {
+		testMessages = append(testMessages, i)
+	}
+	testChunks := chunkMessages(500, testMessages)
+	must.Equal(t, 1, len(testChunks))
+	must.Equal(t, 500, len(testChunks[0]))
+
+	// test that we have everything we expect in the right order and chunks
+	for i, chunk := range testChunks {
+		if i == 0 {
+			for ii := 0; ii < 500; ii++ {
+				must.Equal(t, chunk[ii], testMessages[ii])
+			}
+		}
+	}
```

Comment:
This should do just as well:

```suggestion
	must.Equal(t, testMessages, testChunks[0])
```